drivers/dma/dma_mcux_edma.c

```diff
@@ -48,8 +48,7 @@ struct dma_mcux_edma_data {
 	struct call_back data_cb[DT_INST_PROP(0, dma_channels)];
 };
 
-#define DEV_CFG(dev)                                                           \
-	((const struct dma_mcux_edma_config *const)dev->config)
+#define DEV_CFG(dev) ((const struct dma_mcux_edma_config *const)dev->config)
```

Comment:
Please move the whitespace changes to a separate commit

Reply:
revert back